subsys/net/lib/sockets/sockets.c

```diff
@@ -106,8 +107,11 @@ int _impl_zsock_close(int sock)
 	 * as these are fail-free operations and we're closing
 	 * socket anyway.
 	 */
-	(void)net_context_accept(ctx, NULL, K_NO_WAIT, NULL);
-	(void)net_context_recv(ctx, NULL, K_NO_WAIT, NULL);
+	if (is_listen) {
```

Comment:
No need to have extra variable for this, just place the check directly into if() statement.

Reply:
Well, that's one of the way deal with miniscule line length limit of Zephyr codestyle, to avoid multiline `if`'s. But will fix.

Reply:
Heh, actually in this case no line wrapping was required indeed. Was copy-pasted from existing code...